<launch>
    <node name="controller_models" pkg="niryo_moveit" type="controller_models.py" args="--wait" output="screen" />
</launch>